src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_InvalidExpression.cs

```diff
@@ -33,9 +33,7 @@ static void Main(string[] args)
             string expectedOperationTree = @"
 IInvalidOperation (OperationKind.Invalid, Type: ?, IsInvalid) (Syntax: 'Console.WriteLine2()')
   Children(1):
-      IInvalidOperation (OperationKind.Invalid, Type: ?, IsInvalid) (Syntax: 'Console.WriteLine2')
```

Comment:
>IInvalidOperation (OperationKind.Invalid, Type: ?, IsInvalid) (Syntax: 'Console.WriteLine2') [](http://example.com/codeflow?start=6&length=92)

Do we understand why this node disappeared?

Reply:
Yes, it is due to the change in `tryBindMemberAccessWithBoundTypeLeft` (Binder_Expression.cs line 7916)
See explanation: [https://github.com/dotnet/roslyn/pull/68629#discussion_r1253432174](https://github.com/dotnet/roslyn/pull/68629#discussion_r1253432174)